PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutAccount.java

```diff
@@ -106,6 +108,16 @@ void setPaymentType(String paymentType) {
         this.paymentType = paymentType;
     }
 
+    /**
+     * Used by PayPal wrappers to construct a request to create a PayPal account.
+     *
+     * @param intent The intent provided by the checkout session
+     */
+    void setIntent(String intent) {
+        this.intent = intent;
```

Comment:
Should this be added to the changelog as well?

Reply:
yup I can update it!